features/login.feature

```diff
@@ -36,7 +36,7 @@ Feature: Log in as an admin, as a teacher, as a student
         As a teacher, I want to be able to login
         When teacher logs in with email <teacherUsername> and password <password>
         When teacher accepts data protection
-        Then the teacher-dashboard should have an icon with the teacher's initials
```

Comment:
method used in this step was the same as checkInitials -- so we have to implement NavigationTopPage.checkNameAndProfession(); later 
therfore I commented this step out because it makes no sense to check something twice